otb-tracker/init

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh /etc/rc.common
+# vim: set noexpandtab tabstop=4 shiftwidth=4 softtabstop=4 :
+
+START=90
+STOP=10
+
+USE_PROCD=1
+
+. /lib/otb-tracker
+
+validate_section(){
+	uci_validate_section otb-tracker track "$1" \
+		'host:host' \
+		'timeout:uinteger:1' \
+		'retry:uinteger:2' \
+		'interval:uinteger:2' \
+		'domain:string'
+}
+
+_start_track() {
+	interface="$1"
+	case "$interface" in
+		loopback) return;;
+		lan*) return;;
+		if0*) return;;
+	esac
+	validate_section "default"
+	procd_open_instance
+	procd_set_param command /bin/otb-tracker-dns -t "$timeout" -h "$host" -i "$interface" -r "$retry" -d "$domain" -s "$interval"
+	procd_set_param respawn procd_set_param stderr 1
+	procd_set_param stdout 1
+	procd_close_instance
+}
+
+start_service() {
+	otb_tracker_generate_conf default
+	config_load network
+	config_foreach _start_track interface
+}
+
+service_triggers() {
+	procd_add_reload_trigger otb-tracker
```

Comment:
You can add them all on the same line:
From the [doc](https://wiki.openwrt.org/inbox/procd-init-scripts):
```
service_triggers()
{
        procd_add_reload_trigger "uci-file-name" "second-uci-file"
        procd_add_network_trigger "lan"|"etho0" FIXME - this is still a work in process....
}
```